packaging/test_plugin_unified_storage_tiering.py

```diff
@@ -879,6 +950,19 @@ def test_iput_new_object_does_not_trigger_restage__issue_303(self):
             if os.path.exists(filename):
                 os.unlink(filename)
 
+    def test_replica_open_replica_close_does_not_trigger_restage__issue_316(self):
+        # TODO(#280): This test will fail once tiering group metadata is annotated to objects on create. This test
+        # should continue to pass as-is.
+        filename = "test_replica_open_replica_close_does_not_trigger_restage__issue_316"
+        logical_path = "/".join([self.user1.session_collection, filename])
+        self.creating_data_object_does_not_trigger_restage_test_impl(
+            logical_path,
+            self.tier2,
+            self.user1.assert_icommand,
+            ["istream", "-R", self.tier2, "write", logical_path],
+            "STDOUT",
+            input="do not restage this")
```

Comment:
I feel like I'm having a brain fart on the TODO comment. Please correct me if I'm wrong, but the comment is just saying ... when data objects are **created** and storage tiering metadata is attached to it _(in the future)_, this test will fail. Is that correct?

Reply:
Right. The only reason the test passes right now is because the tier group metadata isn't being annotated on create. Once #280 is fixed, this test will begin to fail and the comment is here to remind us that there's nothing wrong with the test.

Reply:
Got it. Thanks.